clouddriver-kubernetes/src/test/java/com/netflix/spinnaker/clouddriver/kubernetes/caching/view/provider/KubernetesDataProviderIntegrationTest.java

```diff
@@ -160,7 +160,7 @@
   private static KubernetesV2ServerGroupManagerProvider serverGroupManagerProvider =
       new KubernetesV2ServerGroupManagerProvider(cacheUtils);
   private static KubernetesV2ArtifactProvider artifactProvider =
-      new KubernetesV2ArtifactProvider(cacheUtils, objectMapper);
+      new KubernetesV2ArtifactProvider(accountResolver);
```

Comment:
One of my favorite things in the past week is that this was the only change I needed to make to the integration test.

Reply:
💯 💯 💯